### PR TITLE
Add evdev Python package recipe

### PR DESCRIPTION
**Pull request by ehfd**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
@conda-forge-admin, please ping conda-forge/help-python-c, ready for review!

**Comment by ehfd** [comment]
Explanation about this package for reviewers:

1. The PR must be rerendered by the reviewer in order for the build to succeed, as `__glibc=2.28` is required. This is a Linux-only package and is intended to be deployed on x64, aarch64, ppc64le, and s390x.
2. This package interfaces to the Linux kernel `uinput` interface and thus requires the kernel headers "linux/input.h", "linux/input-event-codes.h", "uapi/linux/uinput.h". [`setup.py`](https://github.com/gvalkov/python-evdev/blob/main/setup.py#L18) requires the usage of environment variables `CPATH` or `C_INCLUDE_PATH`.
3. However, the package also requires the `UI_GET_SYSNAME ioctl`. This is only included in Kernel versions 3.15 and above. Therefore, kernel 4.18, leading to sysroot 2.28 has been used.
4. The reason the staged recipes build currently fails is because the test procedures fail to install sysroot 2.28 for some reason. I believe that a rerender is required, which I cannot currently do.

**Comment by ehfd** [comment]
@conda-forge-admin, please ping conda-forge/help-python-c, not ready for review

File: conda-forge.yml [hunk 1]
```diff
@@ -1,5 +1,18 @@
-channel_priority: strict
```
Comment (xhochy) [inline conda-forge.yml:1]: Please don't touch this file.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: conda-forge.yml (hunk 1 above)
Comment (ehfd) [reply conda-forge.yml]: Resolved.

**Comment by ehfd** [comment]
@conda-forge-admin, please ping conda-forge/help-python-c, ready for review!

File: recipes/evdev/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  sha256: 0c72c370bda29d857e188d931019c32651a9c1ea977c08c8d939b1ced1637fde
+
+build:
+  skip: true  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  script_env:
+    - "C_INCLUDE_PATH=${BUILD_PREFIX}/x86_64-conda-linux-gnu/sysroot/usr/include"       # [linux64]
+    - "C_INCLUDE_PATH=${BUILD_PREFIX}/aarch64-conda-linux-gnu/sysroot/usr/include"      # [aarch64]
+    - "C_INCLUDE_PATH=${BUILD_PREFIX}/powerpc64le-conda-linux-gnu/sysroot/usr/include"  # [ppc64le]
+    - "C_INCLUDE_PATH=${BUILD_PREFIX}/s390x-conda-linux-gnu/sysroot/usr/include"        # [s390x]
```
Comment (jaimergp) [inline recipes/evdev/meta.yaml:19]: Aren't these covered by the compiler / sysroot activation scripts? 🤔 Does it work if you remove this?

Comment (ehfd) [reply recipes/evdev/meta.yaml:19]: It did not work. This envvar is parsed in a custom way within setup.py.

Comment (ehfd) [reply recipes/evdev/meta.yaml:19]: https://github.com/gvalkov/python-evdev/blob/0d496bf8a5bce2d5c60147609cb79df1386dbf23/setup.py#L15

Comment (ehfd) [reply recipes/evdev/meta.yaml:19]: @jaimergp

Considering that going down to build_ext is not recommended in pip, this should be the best way.

Comment (ehfd) [reply recipes/evdev/meta.yaml:19]: Precisely what I wanted. Doing right away.

Comment (ehfd) [reply recipes/evdev/meta.yaml]: Perhaps the failing issue is that `activate-binutils_linux-64.sh` is after `script_env`.

**Comment by ehfd** [comment]
@jaimergp Everything is ready.

**Review by jaimergp — APPROVED** [review]

**Pull request merged**